src/main/java/com/sarality/db/UpdateOperationType.java

```diff
@@ -0,0 +1,17 @@
+package com.sarality.db;
+
+/**
+ * enum for list of operations to be supported when a database is upgraded.
+ *
+ * @author Satya@ (Satya Puniani) on 01/03/17
```

Comment:
Can you please edit the new class template and remove the date for new classes that are generated